source/addonHandler/__init__.py

```diff
@@ -76,30 +76,19 @@ def getRunningAddons():
 	"""
 	return getAvailableAddons(filterFunc=lambda addon: addon.isRunning)
 
-def getAddonsWithUnknownCompatibility(version=addonVersionCheck.CURRENT_NVDA_VERSION):
-	""" Returns add-ons that have "UNKNOWN" compatibility state value. These are addons
-	that have no auto-deduced or manually set compatibility state.
-	"""
-	return getAddonsMatchingCompatValues([compatValues.UNKNOWN], version)
-
-def getAddonsWithoutKnownCompatibility(version=addonVersionCheck.CURRENT_NVDA_VERSION):
-	""" Returns add-ons that are untested for the specified NVDA version.
-	"""
-	withoutKnownCompatValues = [
-			compatValues.UNKNOWN,
-			compatValues.MANUALLY_SET_COMPATIBLE,
-			compatValues.MANUALLY_SET_INCOMPATIBLE
-		]
-	return getAddonsMatchingCompatValues(withoutKnownCompatValues, version)
-
-def getAddonsMatchingCompatValues(compatValues, version=addonVersionCheck.CURRENT_NVDA_VERSION):
-	""" Returns add-ons that have compatibility value that is True when anded with compatMask
+def getIncompatibleAddons(
+		currentAPIVersion=addonAPIVersion.CURRENT,
+		backCompatToAPIVersion=addonAPIVersion.BACK_COMPAT_TO):
+	""" Returns a generator of the add-ons that are not compatible.

```

Comment:
The doc string? It now comes right after the function definition, but because the argument names and default values are long they are on a new line each. I could put both the args on one line, but it would be quite long.

Reply:
Ugh, never mind, that's what happens if you review code without braille. I thought the parameters were variable assignments inside the function's body.